code/modules/admin/verbs/possess.dm

```diff
@@ -2,7 +2,7 @@
 	set name = "Possess Obj"
 	set category = "Object"
 
-	if((O.obj_flags & DANGEROUS_POSSESSION) && CONFIG_GET(flag/forbid_singulo_possession))
+	if((O.obj_flags & DANGEROUS_POSSESSION))
```

Comment:
also test your damn code, this would make the singulo permanently impossible to posses, since it still has this flag